plonk/src/circuit/basic.rs

```diff
@@ -264,6 +264,15 @@ impl<F: FftField> PlonkCircuit<F> {
     pub fn range_size(&self) -> Result<usize, PlonkError> {
         Ok(1 << self.range_bit_len()?)
     }
+
+    /// creating a `BoolVar` without checking if `v` is a boolean value!
+    /// You should absolutely sure about what you are doing.
+    /// You should normally only use this API if you already enforce `v` to be a
+    /// boolean value using other constraints.
+    pub(crate) fn create_bool_variable_unchecked(&mut self, a: F) -> Result<BoolVar, PlonkError> {
```

Comment:
slightly prefer `boolean_variable` over `bool_variable`

Reply:
done in [5f0cd66](https://github.com/EspressoSystems/jellyfish/pull/94/commits/5f0cd663417b46145cedbd3f620aa8963808ba00)